### DISCUSSION TITLE
[Bug]: YOLO NAS playbook doesn’t work

### DISCUSSION THREAD

**Discussion by igiannakas** [blakeblackshear/frigate#13684] [Report a Bug] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

When running the yolo nas notebook the following error is reported: 

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
albucore 0.0.14 requires numpy>=1.24, but you have numpy 1.23.0 which is incompatible.
chex 0.1.86 requires numpy>=1.24.1, but you have numpy 1.23.0 which is incompatible.
pandas-stubs 2.1.4.231227 requires numpy>=1.26.0; python_version < "3.13", but you have numpy 1.23.0 which is incompatible.
tensorflow 2.17.0 requires numpy<2.0.0,>=1.23.5; python_version <= "3.11", but you have numpy 1.23.0 which is incompatible.

### Steps to reproduce

1. Click on the playbook link
2. Click run
3. Error is shown
...

### Version

0.14

### Frigate config file

```yaml
N\/A
```

### docker-compose file or Docker CLI command

```yaml
N/A
```

### Relevant Frigate log output

```shell
N/A
```

### Relevant go2rtc log output

```shell
N/A
```

### Operating system

HassOS

### Install method

HassOS Addon

### Network connection

Wired

### Camera make and model

N/A

### Screenshots of the Frigate UI's System metrics pages

N/A

### Any other information that may be helpful

![IMG_4545](https://github.com/user-attachments/assets/ea48a019-7aaf-41f1-b0dc-42e859b69ccf)

**Comment by NickM-27** [comment, marked as answer]
you can still continue on to the next step

**Comment by igiannakas** [comment]
It worked beautifully. Thank you and sorry for the unnecessary question :)
